tests/default.nf.test

```diff
@@ -15,20 +15,136 @@ nextflow_pipeline {
+
+            // porechop_abi
+            def stable_content_porechopabi = getAllFilesFromDir("$outputDir/porechop_abi", relative: true, includeDir: false, ignore: ['*.{log}'] , ignoreFile: null, include: ['*','**/*'])
+            def stable_name_porechopabi  = getAllFilesFromDir("$outputDir/porechop_abi", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*.log'])
+
+            // samtools
+            def stable_name_samtools     = getAllFilesFromDir("$outputDir/samtools", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*'])
+
+            //taxpasta
+            def stable_content_taxpasta  = getAllFilesFromDir("$outputDir/taxpasta", relative: true, includeDir: false, ignore: ['*.{tsv}'], ignoreFile: null, include: ['*', '**/*'])
+            def stable_name_taxpasta     = getAllFilesFromDir("$outputDir/taxpasta", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*.{tsv}'])
+
+
             assertAll(
                 { assert workflow.success},
                 { assert snapshot(
                     // pipeline versions.yml file for multiqc from which Nextflow version is removed because we test pipelines on multiple Nextflow versions
                     removeNextflowVersion("$outputDir/pipeline_info/nf_core_taxprofiler_software_mqc_versions.yml"),
-                    // All stable path name, with a relative path
-                   // stable_name,
-                    // All files with stable contents
-                    //stable_path
-                ).match() }
+                ).match() },
+
+                { assert new File("$outputDir/pipeline_info/nf_core_taxprofiler_software_mqc_versions.yml").exists() },
+                { assert new File("$outputDir/multiqc/multiqc_report.html").exists() },
+
+                { assert snapshot ( stable_name_all ).match("all_files") },
+		            { assert snapshot ( stable_name_multiqc ).match("multiqc" ) },
+                { assert snapshot( file("$outputDir/bbduk/2613_ERR5766181.bbduk.log").text.contains("Reads Processed:"), file("$outputDir/bbduk/2612_ERR5766180.bbduk.log").text.contains("Reads Processed:"),file("$outputDir/bbduk/2612_ERR5766176.bbduk.log").text.contains("Reads Processed:"),file("$outputDir/bbduk/2612_ERR5766176_B.bbduk.log").text.contains("Reads Processed:"),stable_content_bbduk, stable_name_bbduk).match("bbduk")},
+                { assert snapshot ( stable_content_diamond, stable_name_diamond, file("$outputDir/diamond/db2/2614_db2.diamond.log").text.contains("133 queries aligned"),file("$outputDir/diamond/db2/2614_db2.diamond.tsv").text.contains("ERR3201952.3404519"),file("$outputDir/diamond/db2/ERR3201952_db2.diamond.log").text.contains("91 queries aligned"),file("$outputDir/diamond/db2/ERR3201952_db2.diamond.tsv").text.contains("ERR3201952.3889254")).match("diamond" ) },
```

Comment:
Is it correct you've checked two long read samples, and not one long read and one short read sample?